apps/backend/src/app/bids/bids.controller.ts

```diff
@@ -0,0 +1,18 @@
+import { Controller, Get, Post, Body } from '@nestjs/common';
+import { BidsService } from './bids.service';
+import { CreateBidDto } from './dto/create-bid.dto';
+
+@Controller('bids')
+export class BidsController {
+  constructor(private readonly bidsService: BidsService) {}
+
+  @Post('/create')
```

Comment:
I think that **create** word is not necessary in this place. We adopted standard REST Api convension that methods of sending request decides what happening in application. Post method is always responsible for creating resource ;)